src/components/table/_macro.njk

```diff
@@ -16,7 +16,7 @@
                 </thead>
                 <tbody class="table__body">
                     {% for tr in params.trs %}
-                    <tr class="table__row">
+                    <tr class="table__row{{ " table__row--highlight" if tr.highlight }}">
```

Comment:
I think this highlight param needs to be added to the mark down. Other than that looks good